README.md

```diff
@@ -51,14 +59,35 @@ queue.async {
     This task begins execution once the above one-second sleep completes.
     */
 }
-Task {
```

Comment:
This `Task` was meant to show that we were entering an async context, but I'm realizing now that the inclusion of the `Task` is more confusing than not. Nothing within the `Task` closure changed – I just removed the `Task` enclosure. I did the same thing further down as well.

Reply:
This makes sense to me. The reader can infer from the `await` below that we're in an asynchronous context.